### ISSUE TITLE
SIB-FLEURS Classifier architecture

### ISSUE THREAD

**Issue by mdhaffar** [fdschmidt93/fleurs-slu#2]
Hello,

I want to ask about the architecture used to train classification models in the paper FLEURS-SLU for the task SIB-FLEURS.

Thank you,

**Comment by fdschmidt93** [comment]
Our Huggingface repository for the Seamless speech encoder has an example of how to use the it on SIB-Fleurs: https://huggingface.co/WueNLP/seamless-m4t-v2-large-speech-encoder

I hope this answers your question!

**Comment by mdhaffar** [comment]
Thank you for your answer.

I have a follow-up question regarding the results. In the paper, the performance for non-English (ZS-XLT) is reported at 79%, whereas on Hugging Face, it appears as 70%. 
I would greatly appreciate it if you could clarify the reason behind this 9% difference.

**Comment by fdschmidt93** [comment]
The CSV on Huggingface generally has much lower scores, since it is only 3 much lower learning rates {1,2,3}e-5 vs {1,...,10e-5} in the paper. Seamless got the best results with LRs between 6e-5 and 8e-5, with larger learning rates performing much better. The CSV results comprises the results I had at the time as I had to publish the dataset, since Meta put up the preprint for 2M-Belebele.

Here is an updated sheet with full per language results and all relevant meta data by model (uttarance quality, modality, learning rate) for all the languages of SIB and Belebele. Models are selected on the English validation split: https://docs.google.com/spreadsheets/d/1Nf8xiRbuNgHgk4lYs6tKSuJJ20eW6v_4wbRGvpIgc88/edit?usp=sharing

One cautionary note of SIB-Fleurs is that the validation split is quite small. What ends up happening that, if you would "cheat" and look at test performance throughout you would notice that suboptimal models get selected and you could get seemingly significant 1-2% of test performance. That is to shed light on why, for instance, English results for varying utterance quality may appear not perfectly consistent (sometimes worse utterance quality performs better). Ultimately, the take away here is, as written in the paper, that speech and ASR models are just very robust for English. 

cc @helleuch

**Comment by mdhaffar** [comment]
Thank you for this clarification!
